plugins/python_3_script/bin/icon_python_3_script

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env python
-# GENERATED BY KOMAND SDK - DO NOT EDIT
+# GENERATED BY INSIGHT-PLUGIN - DO NOT EDIT
 import os
 import json
 from sys import argv
 
 Name = "Python 3 Script"
 Vendor = "rapid7"
-Version = "4.0.5"
-Description = "Run a Python 3 script"
+Version = "4.0.6"
+Description = "Python is a programming language that lets you work quickly and integrate systems more effectively. This plugin allows you to run Python 3 code. It includes Python 3.6.5 and its standard library as well as the following 3rd party libraries"
```

Comment:
based on the docker file it looks, like you are using python 3.8 rather than python 3.6.5, was there a reason that the description was updated to say 3.6.5?

```
FROM rapid7/insightconnect-python-3-38-plugin:5
```

Reply:
I think that was the old description but good catch, the description needs updated

Reply:
Good catch! fixed it up now